### ISSUE TITLE
weighted loss (different from paper PVN3d)

### ISSUE THREAD

**Issue by minwang-ai** [ethnhe/FFB6D#36]
Hi Yisheng,

You indicated that you followed the previous work PVN3D to set the weights to 1,1,1 in paper FFB6D. In fact, it is 2,1,1. It is a mistake or do you find these weights would lead to better performance?
```
loss_lst = [
                (loss_rgbd_seg, 2.0), (loss_kp_of, 1.0), (loss_ctr_of, 1.0),
            ]
            loss = sum([ls * w for ls, w in loss_lst])
```

**Comment by ethnhe** [comment]
We use 1, 1, 1 for both PVN3D and FFB6D in our internal versions while both use 2, 1, 1 in the open source versions so that their performances are closer. Only with very slight difference though.

**Comment by minwang-ai** [comment]
> We use 1, 1, 1 for both PVN3D and FFB6D in our internal versions while both use 2, 1, 1 in the open source versions so that their performances are closer. Only with very slight difference though.

Hi, Yisheng, thank you for your reply.
What's the meaning of closer performances? 
Is there any other difference between your internal version and the open-source version?

**Comment by LEONHWH** [comment]
> > 我们在内部版本中对 PVN3D 和 FFB6D 都使用 1、1、1，而在开源版本中都使用 2、1、1，以便它们的性能更接近。不过只有很小的区别。
> 
> 你好，易生，谢谢你的回复。 近距离表演是什么意思？ 您的内部版本和开源版本之间还有其他区别吗？

Hi,minwang.

Have you tried these two different weighted losses?
Or, do you know what the best loss weighting factor is?

**Comment by minwang-ai** [comment]
Sorry, I did not try that. I just keep everything as it was since I changed my topic and supervisor.
